modules/RTC/BridgeChannel.js

```diff
@@ -358,11 +359,26 @@ export default class BridgeChannel {
                 break;
             }
             case 'LastNEndpointsChangeEvent': {
-                // The new/latest list of last-n endpoint IDs (i.e. endpoints for which the bridge is sending video).
-                const lastNEndpoints = obj.lastNEndpoints;
+                if (!FeatureFlags.isSourceNameSignalingEnabled()) {
```

Comment:
I've added this check to indirectly stop emitting the `JitsiConferenceEvents.LAST_N_ENDPOINTS_CHANGED` and `JitsiConferenceEvents.PARTICIPANT_CONN_STATUS_CHANGED` legacy events when the feature flag is enabled.

Reply:
yes, that's good